docs/docs/docs/developer-resources/troubleshooting.md

```diff
@@ -98,3 +98,406 @@ mongosh","version":"6.12.0|2.3.8"},"platform":"Node.js v20.18.1, LE","os":{"name
 {"t":{"$date":"2025-02-22T01:14:08.040+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:36848","uuid":{"uuid":{"$uuid":"1ef5fcbd-4913-45fe-bc66-7bc3600a941a"}},"connectionId":2195,"connectionCount":24}}
 {"t":{"$date":"2025-02-22T01:14:08.043+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:36854","uuid":{"uuid":{"$uuid":"48522796-7b00-46df-a5d1-3e2a9ec7edd8"}},"connectionId":2196,"connectionCount":25}}
 ```
+
+
+<!-- add Non Docker Troubleshooting -->
+
+
+## Non-Docker Production Server Troubleshooting
+
+This section covers troubleshooting for Talawa Admin running in a non-Docker production environment.
+
+### Finding and Reading Application Logs
+
+#### Console Output (Direct Execution)
+
+When running the app with `npm start` or `npm run serve`, logs appear directly in the terminal:
+
+```bash
+# Run and save logs to a file
+npm start > app.log 2>&1
+
+# Run in background and save logs
+nohup npm start > app.log 2>&1 &
+
+# View logs in real-time
+tail -f app.log
+```
+
+#### PM2 Logs
+
+PM2 provides comprehensive logging capabilities:
```

Comment:
1. Why was PM2 used?
2. Is PM2 the default with typescript?
3. Add a brief description of what PM2 is.
4. If the logs are located in a `.pm2` directory, then `.pm2` directory needs to be added to the `.gitignore` file